.github/autobuild/linux_deb.sh

```diff
@@ -10,9 +10,17 @@ TARGET_ARCH="${TARGET_ARCH:-amd64}"
 case "${TARGET_ARCH}" in
     amd64)
         ABI_NAME=""
+        # MKSPEC_ABI_REPLACE defines a sed substitution pattern which is applied in setup_cross_compiler
+        # for arm* systems only:
+        MKSPEC_ABI_REPLACE=""
         ;;
     armhf)
         ABI_NAME="arm-linux-gnueabihf"
+        MKSPEC_ABI_REPLACE='s/-gnueabi/-gnueabihf/'
```

Comment:
In fact... this is the same for both:
```suggestion
        MKSPEC_ABI_REPLACE="s/arm-linux-gnueabi/${ABI_NAME}/"
```
which means the variable can actually go.  This is just a "short hand", as the `-gnueabi` is expected to be enough to limit the hit.

Reply:
Heh, nice idea, didn't spot that. Done.

Reply:
Hey, neither did I till now :)